client/packages/common/src/ui/components/modals/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -48,6 +50,13 @@ export const ConfirmationModal = ({
         <Grid item>
           <Typography style={{ whiteSpace: 'pre-line' }}>{message}</Typography>
         </Grid>
+        {info && (
+          <Grid item paddingY={1}>
+            <Alert style={{ whiteSpace: 'pre-line' }} severity="info">
+              {info}
+            </Alert>
+          </Grid>
+        )}
```

Comment:
Per Mariya's comment, if we can move this above the Typography that would be great! (So heading, then alert, then message) 🙏

Reply:
Great, I have moved the alert up above the typography. I have also changed the message so that they wok together better (open to suggestions though!)

![Screenshot 2024-12-17 at 10 05 37 AM](https://github.com/user-attachments/assets/003a28c1-7f81-4d3f-bc73-e84c8c938668)

Reply:
Suggestions for this?
I followed the existing logic for when the alert is to show/not show  - it shows when there is **no** shipment on the requisition. The message shows if there is **any** line that is not fully shipped.
So if one line is shipped and the other isn't, the alert does not display but the message does. 

I'd be happy to link these into the same function so that the alert will appear every time the message is triggered

![Screenshot 2024-12-17 at 10 08 30 AM](https://github.com/user-attachments/assets/0c64b8c4-f261-4437-9ce5-41cd97d4b62c)

Reply:
> I'd be happy to link these into the same function so that the alert will appear every time the message is triggered

Was about to ask for this! Yep I think always showing both (if any lines not fully supplied) would now make the most sense. I actually really like the alert breaking up the text, feels less like you're getting blasted with a paragraph that's too easy to ignore 😁

Reply:
This is tidied up in the code for logic to be consistent - if anything is not supplied the alert and message will both appear

Alert shows between the heading and text with the updated wording

Reply:
@lache-melvin we also discussed removing the extra status logic - when doing this there was a bit relying on it throughout the area and would have been (for me) quite a rewrite. It works and does what it needs to do, so leaving as it is for now.

If it's important enough, I'm happy to look further into it or write up a new issue